src/elements/hx-progress/index.js

```diff
@@ -43,7 +43,9 @@ export class HXProgressElement extends HXElement {
         this.$defaultAttribute('role', 'progressbar');
         this.$defaultAttribute('aria-valuemin', MIN);
         this.$defaultAttribute('aria-valuemax', MAX);
-        this.value = this.value;
+
+        let configuredValue = this.value;
+        this.value = configuredValue;
```

Comment:
correct `no-self-assign` error now showing up with latest `eslint`

Reply:
TODO: revert changes and reconfigure linting rules to disable `no-self-assign` checks.